.gitignore

```diff
@@ -4,3 +4,6 @@
 *.pyc
 config.py
 *.log
+.cache
+env
+lastfm-backup-upload
```

Comment:
If we use the /tmp/lastfm-backup-upload default, then this doesn't need to be here.